data.json

```diff
@@ -1342,5 +1342,11 @@
     "githubUsername": "chetan-2002",
     "imageUrl": "https://user-images.githubusercontent.com/77783033/133980058-5a671e7b-f80a-400d-933e-613118a32e02.png",
     "issueNumber": 724
+  },
+  {
+    "name": "Shreyaans Jain",
+    "githubUsername": "shreyaansjain06",
+    "imageUrl": "https://user-images.githubusercontent.com/86606251/133983008-13e73a8d-fb2b-4592-98b1-7f1cfbbcbb69.png",
```

Comment:
I would recommend more of an image like this.

![image](https://user-images.githubusercontent.com/51391473/134031514-b0bbff29-3974-4f05-a2c6-a5c163e90216.png)

```suggestion
    "imageUrl": "https://user-images.githubusercontent.com/51391473/134031514-b0bbff29-3974-4f05-a2c6-a5c163e90216.png",
```